packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1448,6 +1462,18 @@ class _SelectableRegionContainerDelegate extends MultiSelectableSelectionContain
 /// This class optimize the selection update by keeping track of the
 /// [Selectable]s that currently contain the selection edges.
 abstract class MultiSelectableSelectionContainerDelegate extends SelectionContainerDelegate with ChangeNotifier {
+  /// Creates a [MultiSelectableSelectionContainerDelegate].
+  ///
+  /// The [separator] is used to separate the text of each selectable in
```

Comment:
Nit: This isn't very clear about what it's used for: "used to separate".  Would it be appropriate to mention that it's used when copying text across `selectables`, or does it need to be generic here?